waffle-chai/test/contracts/Events.ts

```diff
@@ -2,59 +2,73 @@ export const EVENTS_SOURCE = `
   pragma solidity ^0.6.0;
 
   contract Events {
```

Comment:
It would probably be better to have a separate contract for testing this new feature specifically